docs/source/theming/about-semantic.md

```diff
@@ -9,6 +9,9 @@ myst:
 
 # About Semantic UI
 
+```{include} /_inc/_semanticui-deprecation.md
```

Comment:
All includes must be relative, not from the root, because when Documentation builds, it needs to be able to find the includes in Volto docs. You will need to adjust all the include paths and rename the file as well.

````suggestion
```{include} ../_inc/_semantic-ui-deprecation.md
````